### PR TITLE
blog(embertimes110): updates for colocating temps

### PR DESCRIPTION
**Pull request by chrisrng**

### CODE REVIEW

File: source/2019-08-09-the-ember-times-issue-110.md [hunk 1]
```diff
@@ -92,13 +92,11 @@ responsive: true
   
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Ember Guides Update for co-located templates ☝️](https://github.com/ember-learn/guides-source/pull/977)
```
Comment (jaredgalanis) [inline source/2019-08-09-the-ember-times-issue-110.md]:
```suggestion
## [Ember Guides Update for Co-Located Templates ☝️](https://github.com/ember-learn/guides-source/pull/977)
```

File: source/2019-08-09-the-ember-times-issue-110.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-author: the crowd
+author: Chris Ng, the crowd
 tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2019
 alias : "blog/2019/08/09-the-ember-times-issue-110.html"
 responsive: true
 ---
 
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
-
+An update for co-located templates in the Guides,
```
Comment (jaredgalanis) [inline source/2019-08-09-the-ember-times-issue-110.md]:
```suggestion
An Update for Co-Located Templates in the Guides,
```

**Review by jaredgalanis — COMMENTED** [review]
Looks great! A few small suggested changes around title casing.

**Review by ijlee2 — APPROVED** [review]
Thanks for covering this article!

**Review by jayjayjpg — APPROVED** [review]
This reads great! Thank you for another :sparkles: writeup this week!

**Pull request merged**